src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -8472,6 +8472,49 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/linked-ruleset':
+    post:
+      x-alfresco-since: "7.3.0"
+      tags:
+        - rules
+      summary: Link a ruleset to another folder
+      desciption: |
+        **Note:** this endpoint is available in Alfresco 7.3 and newer versions.
+        
+        Links a ruleset from another folder to the folder node **nodeId**.
+      operationId: linkRuleset
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#parameters/nodeFolderIdParam'
+        - in: body
+          name: linkedRulesetBodyCreate
+          description: The ruleset to add.
+          required: true
+          schema:
+            $ref: '#/definitions/LinkedRulesetBodyCreate'
+      responses:
+        '201':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/Ruleset',
+        '400':
+          description: |
+            Invalid parameter: **folderId** or **linkedRulesetBodyCreate** is invalid
+        '401':
+          description: Authentication failed
+        '403':
+          description: User does not have permission to create a link to this ruleset
+        '404':
+          description: |
+            **nodeId** or **folderId** does not exist
+        '409':
+          description: |
+            Ruleset already exists on folder **nodeId**
```

Comment:
Do we also get a 409 if the folder has rules rather than a ruleset ? 

In other words, change to something like "Rules or ruleset already exists on folder **nodeId**" ?